### PR TITLE
Updated parameters format list formatting

### PR DESCRIPTION
**Pull request by minusnolldag**
Updated exportKey parameters format list formatting to be inline with the rest of SubtleCrypto docs.

### Description
Fixed exportKey formatting.

### Motivation
Standardize the docs.

### Additional details
N/A

### Related issues and pull requests
N/A

### CODE REVIEW

File: files/en-us/web/api/subtlecrypto/exportkey/index.md [hunk 1]
```diff
@@ -39,16 +39,11 @@ exportKey(format, key)
...
-      - : [PKCS #8](/en-US/docs/Web/API/SubtleCrypto/importKey#pkcs_8) format.
-    - `spki`
-      - : [SubjectPublicKeyInfo](/en-US/docs/Web/API/SubtleCrypto/importKey#subjectpublickeyinfo) format.
-    - `jwk`
-      - : [JSON Web Key](/en-US/docs/Web/API/SubtleCrypto/importKey#json_web_key) format.
+  - : A string value describing the data format in which the key should be exported. It can be one of the following:
+    - `raw`: [Raw](/en-US/docs/Web/API/SubtleCrypto/importKey#raw) format.
+    - `pkcs8`: [PKCS #8](/en-US/docs/Web/API/SubtleCrypto/importKey#pkcs_8) format.
+    - `spki`: [SubjectPublicKeyInfo](/en-US/docs/Web/API/SubtleCrypto/importKey#subjectpublickeyinfo) format.
+    - `jwk`: [JSON Web Key](/en-US/docs/Web/API/SubtleCrypto/importKey#json_web_key) format.
```
Comment (wbamberg) [inline files/en-us/web/api/subtlecrypto/exportkey/index.md:46]:
```suggestion
    - `"raw"`: [Raw](/en-US/docs/Web/API/SubtleCrypto/importKey#raw) format.
    - `"pkcs8"`: [PKCS #8](/en-US/docs/Web/API/SubtleCrypto/importKey#pkcs_8) format.
    - `"spki"`: [SubjectPublicKeyInfo](/en-US/docs/Web/API/SubtleCrypto/importKey#subjectpublickeyinfo) format.
    - `"jwk"`: [JSON Web Key](/en-US/docs/Web/API/SubtleCrypto/importKey#json_web_key) format.
```

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thank you @minusnolldag ! We're not very consistent here, but it's a good practice to quote strings in lists like this.

File: files/en-us/web/api/subtlecrypto/exportkey/index.md (hunk 1 above)
Comment (minusnolldag) [reply files/en-us/web/api/subtlecrypto/exportkey/index.md:46]: I agree but was following your current standard SubtleCrypto API i.e. [generateKey](https://github.com/mdn/content/blob/main/files/en-us/web/api/subtlecrypto/generatekey/index.md?plain=1) line 51 and [importKey](https://github.com/mdn/content/blob/main/files/en-us/web/api/subtlecrypto/importkey/index.md?plain=1) line 33.
Do we change the rest before committing this or just leave it without the quotes?

Comment (wbamberg) [reply files/en-us/web/api/subtlecrypto/exportkey/index.md:46]: Fair enough,  let's just merge this one then :). Thanks for the fixes @minusnolldag !

**Review by wbamberg — APPROVED** [review]
👍

**Comment by wbamberg** [comment]
(If you wanted to add quotes to this or other pages in this API I'll happily review them, but also happy to merge this as is.)

File: files/en-us/web/api/subtlecrypto/exportkey/index.md (hunk 1 above)
Comment (minusnolldag) [reply files/en-us/web/api/subtlecrypto/exportkey/index.md:46]: No worries @wbamberg, always happy to help!

**Pull request merged**